src/app/common/config/config.ts

```diff
@@ -179,6 +179,7 @@ const config = {
     BLOCKS_PER_DAY: 7160,
     OPERATORS_PER_PAGE: 50,
     BLOCKS_PER_YEAR: 2613400,
+    BUFFER_VALIDATORS_COUNT: 5,
```

Comment:
Please rename it to `OPERATOR_VALIDATORS_LIMIT_PRESERVE`

Reply:
+